core/src/main/java/com/amplifyframework/predictions/models/LanguageType.java

```diff
@@ -145,11 +158,10 @@
      * @return An enum value of matching language code
      */
     public static LanguageType from(String languageCode) {
```

Comment:
It's likely that this will only be called occasionally, for ~one language code. In which case, it might be better optimize the disk use, at the cost of (occasionally) doing some extra computations.

Storing the code map would be a good choice if you need to do a lot of lookups all the time. But, I don't think that's the case here.

For that reason, the other enums have the `for (EnumType value : values()) ` loops that do matching and return a match if found.

Reply:
Ah good point. I'll do that.